### PR TITLE
Feature/uh2001 create main page

### PR DESCRIPTION
**Pull request by ErSantis**
Create promos banner
image

Create grid products
image

Create final banner
image

### CODE REVIEW

**Comment by ChristianDj456** [comment]
La pagina tiene un diseño atractivo, y hay una buena estructura para el CSS. Buen trabajo

**Review by jfbenitezz — APPROVED** [review]
Well adjusted and organized, cards hover effect is clear, cards could be smaller in the mobile version

**Comment by ErSantis** [comment]
Sorry I forgot to attach images of how it works with responsive. In mobile phone it goes from 3 to two columns for a better visulizacion and the texts change size so that they do not overflow.
image

**Review by imcarlosmsX — APPROVED** [review]
buen trabajo! excelente el design y el responsive.

File: styles.css [hunk 1]
```diff
@@ -1,17 +1,17 @@
 /* GENERAL STYLES */
 body {
     margin: 0;
-    font-family: 'Poppins', sans-serif;
+    font-family: Arial, sans-serif;
```
Comment (ChristianDj456) [inline styles.css:4]: Recomiendo eliminar esta linea, y solo dejar la de arriba, que es la fuente que se utilizara

**Pull request merged**